CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Enabled the SpatialOS toolbar for MacOS.
 - Added a menu item to push additional arguments for iOS devices.
 - Improved workflow around schema generation issues and launching local builds. A warning will now show if attempting to run a local deployment after a schema error.
+- Added the `bAsyncLoadNewClassesOnEntityCheckout` setting to SpatialGDKSettings that allows loading new classes asynchronously when checking out entities. This is off by default.
```

Comment:
This feature sounds great, why would we turn it off by default? (Are there trade-offs?)

Reply:
Async loading is off in native Unreal by default, so I felt it should be up to the developer to enable this.